container_workflow_tool/git_operations.py

```diff
@@ -135,6 +139,8 @@ def show_git_changes(self, tmp, components=None, diff=False):
             tmp (str): Path to the directory that is used to store git repositories
             components (list of str, optional): List of components to show changes for
             diff (boolean, optional): Controls whether the method calls git-show or git-diff
+            branch_name (str, optional): In case of gitlab, forked_branch and branch_name has to be defined.
```

Comment:
`forked_branch` is mentioned multiple times in this file but there is no such variable or function argument.

Reply:
Fixeds by https://github.com/sclorg/container-workflow-tool/pull/71/commits/f62abfafd01a520ef7d759971344db80b36a0ce6